scripts/run-tests.sh

```diff
@@ -1,10 +1,5 @@
-#!/usr/bin/env bash
-# Bash3 Boilerplate. Copyright (c) 2014, kvz.io
-
-set -o errexit
```

Comment:
I think we missed this. Please add this in a followup PR. 
Looking it up, looks like `set -ev`  is the same as `set -o errexit` (exit when a command fails) and `set -o xtrace` (trace commands) but is missing pipefail and checking for undeclared variables.  (See https://kvz.io/blog/2013/11/21/bash-best-practices/ for details)